generating_an_expression_matrix/README.md

```diff
@@ -19,170 +21,355 @@ We'll be using pre-prepared example inputs
+Alevin collapses the steps involved in dealing with dscRNA-seq into a
+single process, so much of what we'll cover is 'plumbing' of inputs, and
+interpretation.  
 
 ### 1. Generate a transcriptome index
 
-Normally, you would need to generate an index for your transcriptome of interest, containing all sequences likely to be present in your experiment's read data. Generating that index is too time-consuming to do as part of this sesssion, so we'll using using one we made earlier. But have a look at the fasta format sequences, which are the same as the ones we used to build the index. 
+A transcriptome index contains all of the transcript sequences and
+corresponding transcript names likely to be present in your experiment's
+read data. Normally, you would need to first generate the transcript
+sequence list and then align these to their corresponding transcript
+names or identifiers, thus generating the transcriptome index. Is there
+a general transcriptome index for each species? Of course! Without
+delving further, however, note that not all transcriptome indices of the
+same species are, in fact, the same! However, generating a custom
+transcriptome index is too time-consuming to do as part of this session,
+so we'll use one a built-in index. But do have a look at the fasta
+format sequences provided, which are the same as the ones we used to
+build the index.   
 
 ### 2. Generate a transcript to gene mapping
 
-To generate gene-level quantifications based on transcriptome quantification, Alevin and similar tools require a conversion between transcript and gene identifiers, which we can derive from the annotations available in genome resources such as Ensembl. The transcripts in this list need to match the ones we used in the sequences reference. The study we're working with here added spike-ins to their samples, so when we created the index above in preparation for this tutorial, we combined sequence information for the biological sequences and the spike-ins. So now we need to generate a transcript to gene mapping with both types of sequence.
-
-In your example data you will see 'Homo_sapiens.GRCh38.95.gtf.gz', which is the human reference annotations as retrieved from Ensembl in GTF format, and a transcript-level description of the ERCC spike-ins (also in GTF format). We will use these to generate the transcript/ gene mapping, first by combining the two types of annotation and second by passing that information to a tool that extracts just the identifiers we need:
+Gene-level, rather than transcript-level, quantification is standard in
+scRNA-seq, which means that that alternatively spliced RNA molecules
+count as the same gene. In a nutshell, this quantification is used
+because:
+
+ * droplet-based scRNA-seq use short reads, which can make it impossible
+to distinguish transcript isoforms; and  
+ * even in full-length single-cell approaches (i.e. smart-seq) counting
```

Comment:
No, actually quantifying at the transcript level INcreases quantification accuracy at the gene level- see e.g. https://f1000research.com/articles/4-1521, since you're using something like the actual length of the molecules involved.

Reply:
Wait, if that's the case why do we use gene-level quantification for smart-seq approaches?

Reply:
People mostly don't find individual transcript readouts very useful- they can be hard to interpret. But again, using transcript-level quantifications to start with improves the gene-level quantification.